anomaly-detection/README.MD

```diff
@@ -0,0 +1,224 @@
+
+# Event pattern detection with Apache Flink and Pravega
+
+
+Sample application which simulates network anomaly intrusion and detection using Apache Flink and Apache Pravega.
+
+This application is based on [streaming-state-machine](https://github.com/StephanEwen/flink-demos/tree/master/streaming-state-machine) which is slightly extended to demonstrate Pravega/Flink integration capabilities.
+
+Events in streams (generated by devices and services, such as firewalls login-, and authentication services) are expected to occur in certain patterns. Any deviation from these patterns indicates an anomaly (attempted intrusion) that the streaming system should recognize and that should trigger an alert.
```

Comment:
" (generated by devices and services, such as firewalls login-, and authentication services) " - awkward phrase